src/error.rs

```diff
@@ -0,0 +1,134 @@
+use std::error::Error as StdError;
+use hyper::StatusCode;
+use std::fmt;
+
+
```

Comment:
Noticed extra blank lines here.  Not sure if that's typical for Rust, but my team generally disallows it, favoring comment blocks to break up sections.

Reply:
Probably left those in after some unused import cleanup. They are gone now.